lib/tasks/sumofus.rake

```diff
@@ -0,0 +1,51 @@
+require 'open-uri'
+
+namespace :sumofus do
+  desc 'Import legacy actions into your database from a specified file'
+  task seed_legacy_actions: :environment do
+    if ARGV[1].nil?
+      abort('Requires a valid url to a file containing the legacy actions to seed.')
+    end
+
+    if ARGV[2].nil?
+      abort('Requires a valid url to a file containing a default header image to attach to the page.')
+    end
+
+    file_handle = open(ARGV[1])
+    data = file_handle.read
+    file_handle.close
+
+    image_handle = open(ARGV[2])
+
+    parsed_data = JSON.load data
+    count = 0
+    layout_id = LiquidLayout.where(title: 'Petition With Small Image').first.id
+    parsed_data.each_pair do |k, entry|
```

Comment:
The `count` var isn't really useful; it's mostly just there as a sanity check to give me an idea of how many pages are thrown in and confirm that we've made it to the end of the script.